docs/sources/datasources/tempo/_index.md

```diff
@@ -125,27 +126,27 @@ Each linked query consists of:
   Interpolate tags using the `$__tags` keyword.
   For example, when you configure the query `requests_total{$__tags}`with the tags `k8s.pod=pod` and `cluster`, the result looks like `requests_total{pod="nginx-554b9", cluster="us-east-1"}`.
 
-### Configure service graph
+### Service Graph
 
 The **Service Graph** section configures the [Service Graph](/docs/tempo/latest/grafana-agent/service-graphs/) feature.
 
 Configure the **Data source** setting to define in which Prometheus instance the Service Graph data is stored.
 
-To use the service graph, refer to the [Service graph section]({{< relref "#use-the-service-graph" >}}).
+To use the Service Graph, refer to the [service graph documentation]({{< relref "#use-the-service-graph" >}}).
```

Comment:
If Service Graph is a proper noun, then we should be consistent in capitalization. 
```suggestion
To use the Service Graph, refer to the [Service Graph documentation]({{< relref "#use-the-service-graph" >}}).
```